tenants/ai-example/single-model-serving-vllm/base/kustomization.yaml

```diff
@@ -3,31 +3,9 @@ kind: Kustomization
 
 namespace: ai-example-single-model-serving
 
-resources:
-  - ./setup-jobs
-  - data-connection.yaml
-  - inference-service.yaml
-  - serving-runtime.yaml
-
-patches:
-  - path: patch-model-details.yaml
-    target:
-      kind: Job
-      name: upload-model-to-s3-vllm
-
-replacements:
-  - source:
-      kind: Secret
-      name: aws-connection-vllm
-      fieldPath: metadata.name
-    targets:
-      - select:
-          kind: Job
-          name: create-buckets-vllm
-        fieldPaths:
-          - spec.template.spec.containers.0.envFrom.0.secretRef.name
-      - select:
-          kind: Job
-          name: upload-model-to-s3-vllm
-        fieldPaths:
-          - spec.template.spec.containers.0.envFrom.0.secretRef.name
+helmCharts:
+- name: vllm-kserve
+  releaseName: granite
+  version: 0.3.7
+  repo: https://redhat-ai-services.github.io/helm-charts/
```

Comment:
Love Helm charts, but they do make it a bit tricky when installing on a disconnected (air-gapped) cluster that doesn't have access to the Internet, such at a customer location...

Reply:
Fair point.  It would probably be good to get some notes into the disconnected section around helm charts.

There are multiple strategies we can employ in disconnected environments.  First off, you can always just fork/copy the code for the helm chart into the customers environment.  That shouldn't be any greater barrier to entry than what we are already expecting if we want to use the AI Accelerator.

Alternatively, most registry tools like artifactory/nexus have helm registry capabilities where you can point the charts at a registry and mirror them.

Helm also has the ability to publish charts directly to an OCI registry so in the future we could potentially migrate our helm-charts repo to publish directly to quay as well.